src/main/java/uk/gov/companieshouse/search/api/config/ElasticSearchConfig.java

```diff
@@ -7,19 +7,31 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import uk.gov.companieshouse.environment.EnvironmentReader;
+import uk.gov.companieshouse.search.api.exception.EndpointException;
+
+import java.net.MalformedURLException;
+import java.net.URL;
 
 @Configuration
 public class ElasticSearchConfig {
 
     @Autowired
     private EnvironmentReader environmentReader;
 
-    private static final String HOST_NAME = "SEARCH_API_HOST";
+    private static final String END_POINT = "END_POINT";
 
     @Bean(destroyMethod = "close")
-    public RestHighLevelClient client() {
+    public RestHighLevelClient client() throws EndpointException {
+
+        URL endpoint;
+        try {
+            endpoint = new URL(environmentReader.getMandatoryString(END_POINT));
+        } catch (MalformedURLException e) {
+            throw new EndpointException("A malformed url has occurred - url may have failed to parse or invalid protocol provided");
```

Comment:
It would be useful to log the expected format for the environment variable along with the error message to clarify where the problem may be, for example:

```java
throw new EndpointException(END_POINT + " environment variable is malformed; expected format is <protocol>://<host>[:port]");

```